package.json

```diff
@@ -11,23 +11,31 @@
     "build:css": "lessc ./src/css/custom-widget.less ./dist/custom-widget.css",
     "build:cssmin": "cleancss -o ./dist/custom-widget.min.css ./dist/custom-widget.css",
     "build": "npm run build:scripts && npm run build:css && npm run build:cssmin",
-    "prepare": "npm run build"
+    "prepare": "npm run build",
+    "test": "vitest",
+    "coverage": "vitest run --coverage",
+    "check:types": "tsc --noEmit"
```

Comment:
Added some scripts to match other projects with tests and coverage using Vitest.